linkerd/docs/config.md

```diff
@@ -283,3 +283,6 @@ Key | Default Value | Description
 noDelay | true | If set to true, enables the use of `TCP_NODELAY` on a socket interface
 reuseAddr | true | If set to true, enables the `SO_REUSEADDR` option
 reusePort | false | If set to true, enables the `SO_REUSEPORT` option, which can be used to bind another Linkerd process to the same interface port.
+readTimeoutSeconds | Unbounded | Configures this client or server with given transport-level socket read timeout.
```

Comment:
s/Unbounded/unbounded

Reply:
I also think we should shorten the new options. For example, we could shorten `readTimeoutSeconds` to `readTImeoutSecs` to keep it consistent with other configuration field names.